### ISSUE TITLE
Add INSDC identifiers to NMDC studies on `insdc_bioproject_identifiers` slot

### ISSUE THREAD

**Issue by sujaypatil96** [microbiomedata/nmdc-runtime#557]
There is data for a number of studies in the NMDC. For a list of all the studies. see here: https://api.microbiomedata.org/docs#/find/find_studies_studies_get

One of the requirements as part of the NCBI Export squad is the need to have the `insdc_bioproject_identifiers` slot on the Study class objects populated. Reason being, we need to know from the presence/absence of information on this slot whether or not we need to create a new BioProject in NCBI.

This issue is requesting that we populate the `insdc_bioproject_identifiers` slot for all studies in NMDC with the shared INSDC identifiers if they are present.

**Comment by shreddd** [comment]
can this be closed?

**Comment by sujaypatil96** [comment]
Yes, this issue can be closed. If we notice that an NMDC study doesn't have any asserted values on/for `insdc_bioproject_identifiers` then, we should confirm this by looking at GOLD, NCBI, etc. and making sure that it indeed doesn't exist. If it does, then make an issue to populate/assert a value for `insdc_bioproject_identifiers` on that study.
